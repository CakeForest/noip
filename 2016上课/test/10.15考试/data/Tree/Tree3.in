27
4 8
26 27
6 12
16 14
12 1
20 16
4 20
23 3
12 17
8 7
4 9
12 22
20 6
4 24
16 5
2 13
24 10
20 26
4 11
8 15
12 21
20 18
26 23
14 19
8 25
20 2
46
23 16
2 4
24 12
24 4
12 6
24 8
4 6
16 4
12 24
20 26
8 12
26 24
7 20
26 12
12 6
14 20
16 24
4 25
12 24
8 24
8 20
16 7
10 4
4 16
2 8
10 8
20 20
20 12
26 16
12 26
6 16
20 12
16 20
16 5
3 10
20 8
8 8
8 16
16 12
10 12
4 2
4 4
24 8
4 16
20 25
4 8
