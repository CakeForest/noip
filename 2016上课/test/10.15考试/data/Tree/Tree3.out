0
16
12
0
0
21
12
14
12
0
12
0
0
0
0
2
0
3
12
21
5
5
1
14
0
0
27
1
19
0
17
1
0
0
14
5
27
0
0
0
16
27
21
14
0
0
